GoogleForCreators/web-stories-wp
# Stickers: Test out Behavior in Page Layouts

## Context
We want stickers to appear properly in page layouts

## Summary
Originally we were considering making stickers a media type. 
Since we went with more of a shapes approach, they should **just work** ™️ 

This PR updates the second page of the beauty template to utilize a sticker for the hearts instead of multiple shapes. Also updates this page to be purely editor supported effects for animations.

## Relevant Technical Choices
NA

## To-do
NA

## User-facing changes
Animations and hearts should be updated on the second page of the beauty template. Those updates should be visible in the templates view as well as in the page layouts within the editor:
<img width="312" alt="Screen Shot 2021-03-26 at 11 57 06 AM" src="https://user-images.githubusercontent.com/35983235/112675607-fed4db80-8e2c-11eb-913c-cba0d95c975a.png">



## Testing Instructions
View the following page in both the template view and in the page layouts pane within the editor. Verify that it looks nice.
<img width="312" alt="Screen Shot 2021-03-26 at 11 57 06 AM" src="https://user-images.githubusercontent.com/35983235/112675694-190eb980-8e2d-11eb-9813-ef31da82a66f.png">



### QA
Follow testing instructions above.





This PR can be tested by following these steps:

1.

### UAT







This PR can be tested by following these steps:

1.

## Reviews

### Does this PR have a security-related impact?



### Does this PR change what data or activity we track or use?



### Does this PR have a legal-related impact?



## Checklist




---



Fixes #6032
